examples/default_app.rs

```diff
@@ -1,10 +1,9 @@
-extern crate mac_notification_sys;
 use mac_notification_sys::*;
 
 fn main() {
     // not setting the application bundle here, should use default
     // let bundle = get_bundle_identifier_or_default("use_default");
     // set_application(&bundle).unwrap();
-    
-    send_notification("title", &None, "message", &None).unwrap();
+
+    send_notification("title", None, "message", None).unwrap();
```

Comment:
Yeah, but I guess no matter what this PR is going to be a breaking change since I moved the `sound` option to be part of the `NotificationOptions`. I assumed passing in references to `Option`s didn't make much sense, tho their content still is a reference.

Reply:
yes, that was not super seriously on my part, just ignore it, its fine